docs/5-operator-manual/0-set-up-and-install.md

```diff
@@ -7,50 +7,51 @@
 
 ### Zarf Deployed K3s Cluster
 
 <!-- TODO: Some duplicated information from the 'Common CLI Uses' page incoming... -->
 
-Now that you have a Zarf CLI to work with, let's make a cluster that you can deploy to! The [init package](../4-user-guide/2-zarf-packages/3-the-zarf-init-package.md) comes with a built-in k3s cluster that you can deploy if you don't already have another cluster available. You can find the relevant [init package release](https://github.com/defenseunicorns/zarf/releases) on the GitHub releases page.
+Once the Zarf CLI is installed, you can create a cluster using the built-in K3s cluster available in the init package (if another cluster isn’t already available). You can find the relevant [init package release](https://github.com/defenseunicorns/zarf/releases) from the GitHub releases page.
 
-Once downloaded, you can install the init package by navigating to the directory containing the init package and running the command [zarf init](../4-user-guide/1-the-zarf-cli/100-cli-commands/zarf_init.md). Zarf will prompt you, asking if you want to deploy the k3s component, you can type `y` and hit enter to have Zarf startup a local single node k3s cluster on your current machine. Other useful information about initializing a cluster with Zarf is available in the [Initializing a Cluster](./0-set-up-and-install.md#initializing-) section later on in this page.
+Once downloaded, you can install the init package by navigating to the directory containing the package and execute the command [zarf init](../4-user-guide/1-the-zarf-cli/100-cli-commands/zarf_init.md) to install it. Zarf will prompt you with a question, asking if you want to deploy the K3s component. Respond by typing "y" and pressing "Enter" to startup a local single node K3s cluster on your current machine. Additional information regarding initializing a cluster with Zarf is provided in the [Initializing a Cluster](./0-set-up-and-install.md#initializing-) section later on in this page.
 
 :::note
-Depending on the permissions of your user, if you are installing k3s through the `zarf init` command you may need to run the command as a privileged user. This can be done by either:
+Depending on the permissions of your user, if you are installing K3s through the `zarf init` command you may need to run the command as a privileged user. To accomplish this, you can:
 
-1. Becoming a privileged user via the command `sudo su` and then running all the Zarf commands as you normally would.
-2. Manually running all the Zarf commands as a privileged user via the command `sudo {ZARF_COMMAND_HERE}`.
-3. Running the init command as a privileged user via `sudo zarf init` and then changing the permissions of the `~/.kube/config` file to be readable by the current user.
-   :::
+1. Become a privileged user by running the command `sudo su` and then executing all the Zarf commands as usual.
+2. Manually run each Zarf command as a privileged user via the command `sudo {ZARF_COMMAND_HERE}`.
```

Comment:
```suggestion
2. Manually run each Zarf command as a privileged user via the command `sudo {ZARF_COMMAND_HERE}`.
```
This actually won't work for the k3s component - it will stall out looking for the cluster because of the kubeconfig location